content/legacy/_pub_2001_03_gui.md

```diff
@@ -47,7 +47,7 @@ First, we'll need to install both Perl and the Tk libraries. If you are using Li
 
 ### Installing ActiveState Perl for Win32
 
-To install the latest version of Perl for Win32, go to the ActiveState download page ([www.activestate.com/Active-Perl/download.htm](http://www.activestate.com/Active-Perl/download.htm)). For Windows 98 and Windows NT, there are no special instructions; download the latest version (APi517e.exe as of this writing), and run the self-installer by double clicking on the file. For Windows95, however, you need to make sure that you have DCOM installed (it's already installed on Win98 and NT machines). The DCOM files and installation instructions are available from the Microsoft's Web site at [www.microsoft.com/com/dcom/dcom95/dcom1\_3.asp](http://www.microsoft.com/com/dcom/dcom95/dcom1_3.asp).
+To install the latest version of Perl for Win32, go to the ActiveState download page ([www.activestate.com/Active-Perl/download.htm](http://www.activestate.com/Active-Perl/download.htm)). For Windows 98 and Windows NT, there are no special instructions; download the latest version (APi517e.exe as of this writing), and run the self-installer by double clicking on the file. For Windows95, however, you need to make sure that you have DCOM installed (it's already installed on Win98 and NT machines). The DCOM files and installation instructions are available from the Microsoft's Web site at [www.microsoft.com/com/dcom/dcom95/dcom1\_3.asp](http://web.archive.org/web/20180121215533/http://www.microsoft.com:80/com/dcom/dcom95/dcom1_3.asp).
```

Comment:
This should be updated to https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-dcom/4a893f3d-bd29-48cd-9f43-d9777a4415b0 instead of the Wayback Machine